src/material/button/_button-theme.scss

```diff
@@ -38,8 +33,44 @@
   ));
 }
 
+@function _get-text-button-color-tokens-for-palette($palette) {
+  @return (
+    focus-label-text-color: theming.get-color-from-palette($palette),
```

Comment:
Shouldn't these tokens be in the tokens file like they were done in the checkbox? https://github.com/angular/components/pull/26744#discussion_r1130720783

Reply:
The `color=primary|accent|warn` APIs aren't supported by the tokens MDC give us (because they just have tokens like label-color, not label-color-primary, label-color-accent, label-color-warn). For now I've been deferring the decision on what to do about these APIs. (We might choose to get rid of them, add custom tokens to support them, give users separate mixins to apply themselves, etc.) 

So for now I've just been adding some extra styles that keeps them working, and for the checkbox since the default was accent, an easy to do that was to just call the standard function to get the color tokens with a theme that replaced accent with one of the other palettes.

The button works a bit differently because the default is actually "unthemed" which draws its values from the foreground palette who's structure is not analogous to the color palettes, so that same trick doesn't work.

Anyways both the checkbox version and this code are more of just temporary things until we figure out what we want to do with these APIs in the new token world